bin/functions.sh

```diff
@@ -1,5 +1,6 @@
 # default settings
 AWSCLI="/usr/bin/aws"
+#AWSCLI_ENDPOINT_URL_OPTION="--endpoint-url <YourEndpointUrl>"
 AWSCLI_COPY_OPT="s3 cp"
 AWSCLI_LIST_OPT="s3 ls"
 AWSCLI_DEL_OPT="s3 rm"
```

Comment:
Since `AWSCLIOPT` is already provided, I think you can do the same thing by using this environment variable without adding `AWSCLI_ENDPOINT_URL_OPTION`.

Unfortunately, currently `AWSCLIOPT` is not documented...

Reply:
Sure. This would be possible. But I thought it would be cleaner to provide an extra var for this option. ❔